docs/migration-guide/to-15.md

```diff
@@ -36,30 +36,60 @@ Additionally, you may no longer need to extend [Prettier's Stylelint config](htt
 }
 ```
 
-If you want to continue using Stylelint to enforce stylistic consistency, you can [migrate the deprecated rules you need to a plugin](../developer-guide/plugins.md). We will remove the rules from Stylelint in the next major release.
+We've removed the [deprecated rules](../user-guide/rules.md#deprecated) from the latest version of the [standard config](https://www.npmjs.com/package/stylelint-config-standard). It still helps you write consistent CSS by turning on many of the [other rules that enforce conventions](../user-guide/rules.md#enforce-conventions), e.g. most of the [`*-notation`](../user-guide/rules.md#notation), [`*-pattern`](../user-guide/rules.md#pattern) and [`*-quotes`](../user-guide/rules.md#quotes) rules.
+
+There are lots of other rules we don't turn on in the [standard config](https://www.npmjs.com/package/stylelint-config-standard) and you can learn more about using them to customize Stylelint to your exact needs in our [new guide](../user-guide/customize.md).
+
+If you want to continue using Stylelint to enforce stylistic consistency, you can [migrate the deprecated rules you need to a plugin](../developer-guide/plugins.md).
 
 ## Breaking changes
 
 Three breaking changes may also affect you:
 
+- removed `processors` configuration property
 - removed support for Node.js 12
-- removed support for processors
 - changed `overrides.extends` behavior
 
+### Removed `processors` configuration property
+
+Processors were our first attempt to support containers of CSS, e.g. Markdown, HTML and CSS-in-JS. We later introduced [custom syntaxes](../developer-guide/syntaxes.md) to fix some of the shortcomings of processors, e.g. incompatibility with the [`--fix` option](../user-guide/options.md#fix).
+
+We've also deprecated the [`postcss-css-in-js`](https://www.npmjs.com/package/@stylelint/postcss-css-in-js) custom syntax. It was not possible to maintain a monolithic custom syntax that attempted to support every CSS-in-JS library, as there are so many of them and each with variations in syntax.
```

Comment:
[note] This is contextual, and we should use this opportunity to deprecate the monolithic custom syntax - a process we kicked off a year ago.

Reply:
Agreed. In addition, we should update the current inference to remove `@stylelint/postcss-css-in-js`:

https://github.com/stylelint/stylelint/blob/f129f3b992accbf37c7d532d0d020c62c4f3749a/lib/getPostcssResult.js#L121-L136

I can open a new pull request if there are no objections. 💪🏼